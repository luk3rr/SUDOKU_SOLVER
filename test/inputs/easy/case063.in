910000203 500000006 400205000 007023900 100008507 000000300 000600080 000000001 650040000 
