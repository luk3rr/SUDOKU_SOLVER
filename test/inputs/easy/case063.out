Solving the following grid:
9 1 0 | 0 0 0 | 2 0 3 
5 0 0 | 0 0 0 | 0 0 6 
4 0 0 | 2 0 5 | 0 0 0 
------+-------+------
0 0 7 | 0 2 3 | 9 0 0 
1 0 0 | 0 0 8 | 5 0 7 
0 0 0 | 0 0 0 | 3 0 0 
------+-------+------
0 0 0 | 6 0 0 | 0 8 0 
0 0 0 | 0 0 0 | 0 0 1 
6 5 0 | 0 4 0 | 0 0 0 


Solution found :')

9 1 8 | 4 7 6 | 2 5 3 
5 7 2 | 1 3 9 | 8 4 6 
4 3 6 | 2 8 5 | 1 7 9 
------+-------+------
8 6 7 | 5 2 3 | 9 1 4 
1 4 3 | 9 6 8 | 5 2 7 
2 9 5 | 7 1 4 | 3 6 8 
------+-------+------
3 2 1 | 6 9 7 | 4 8 5 
7 8 4 | 3 5 2 | 6 9 1 
6 5 9 | 8 4 1 | 7 3 2 

Algorithm: BFS
Total time: 57 ms
Total expanded states: 9001
